spec/javascript/support/puppeteer/steps/supplementalApplicationSteps.js

```diff
@@ -8,12 +10,32 @@ const testStatusModalUpdate = async (page) => {
   const newSelectedStatus = await page.$eval('.form-modal_form_wrapper .dropdown-menu li[aria-selected="false"] a', e => e.textContent)
   await page.click('.form-modal_form_wrapper .dropdown-menu li[aria-selected="false"] a')
 
-  await checkForSubStatus(newSelectedStatus, page)
+  let selectedSubstatus = await checkForSubStatus(newSelectedStatus, page)
 
   // Enter a comment into the status modal comment field
   await page.type('#status-comment', COMMENT)
   // Submit the status modal form
   await page.click('.form-modal_form_wrapper button.primary')
+
+  // Verify that no form-field errors are present on save
+  const errors = await page.$$eval('span.error', divs => divs.map(d => d.textContent))
+  expect(errors).toStrictEqual([])
+  // Verify that no salesforce save errors present on save
+  try {
```

Comment:
@LoganArnett I'm having a hard time getting this to work. 

My goal is to have a useful error that pops up if there's an issue saving substatuses, ideally with the specific status and substatus that's been sent logged. 

The issue is that asserting that something isn't present while allowing for some time to get a response isn't working great

Reply:
I am not sure if this is the best way to handle this. Another option would be to use another add-on that can actually listen to `http` requests. Something like Sinon gives you the ability to do something like:
```
page.on('response', response => {
    const endpoint = response.url().split('/').pop();
    if (endpoint === "/update_status") {
        const responseBody = await response.json();
        console.log(responseBody)
        // Here we can decipher whether it was an error or not form the API side
    }
})

Reply:
This is the simpler smaller option that is available through Puppeteer but Sinon would allow us to mock and digest more API requests if we see a need for it

Reply:
@LoganArnett what's the simpler smaller option?